src/zalsa.rs

```diff
@@ -393,6 +393,7 @@ where
     phantom: PhantomData<fn() -> I>,
 }
 
+#[allow(clippy::undocumented_unsafe_blocks)] // TODO(#697) document safety
 unsafe impl<I> Sync for IngredientCache<I> where I: Ingredient + Sync {}
```

Comment:
is this impl even necessary? Looking at the definition it should hold by auto trait rules already if I am not mistaken

Reply:
You're right, I'll remove it.